Source/Blazorise/JsOptions/ColorPickerJsOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace Blazorise;
+
+public record ColorPickerJsOptions(
```

Comment:
I would prefer to not use primary constructors. IMO they don't provide much pros, and now we need to be careful about the order of providing arguments when initializing the object. 

So, use regular properties and assign them as before. `PropName = SomeName1`.

Reply:
**Aside Note:** I think you meant to say that you prefer not to use constructors at all **for these enhancements** in particular. They do provide **VERY VERY valuable pros**. But you prefer regular assignment here for readability.

Reply:
Yes. At least not for large option objects. The smaller OptionChange is OK.

Reply:
Ok, just stating it clearly, I know we are in similar positions when it comes to coding practices, but someone else reading your comment about primary constructors might take it the wrong way.

Reply:
I agree with the argument of readability and order changes. The named arguments, as @David-Moreira pointed out, solve this nicely. The advantage over the property assignment is the compiler check. If you add another property to the `ColorPickerJsOptions`, it will scream at you to initialize it too...

By the way, these aren’t primary constructors but rather positional records. The key difference is in how the value is represented: in records, it's a property, while in primary constructors, it's simply a private field (which is ignored by json serializer).

Reply:
One could argue that the named arguments would help for readability, but then we basically have the same thing in the end, just a little bit different syntax

`Propertry1: Value1` vs `Argument1 = Value1`.

Reply:
I don't understand your point. I actually prefer this way if readability was the concern.

Reply:
I personally don't care about using the primary constructor or not, it could be a regular constructor, BUT I prefer the constructor over the loose property assignment, you know... 
1. enforcement of arguments
2. encapsulation / proper access modifiers
3. opportunity to do special centralized init logic on ctor

Reply:
but at the end of the day I prefer consistency, so just choose one way and stick with it in my opinion.